### DISCUSSION TITLE
Custom library error

### DISCUSSION THREAD

**Discussion by bigcat1001** [milaboratory/mixcr#1561] [Q&A] [answered]
When i tried build Custom library,I encountered two errors:
1.The first error is:
```
Sequence dropped because contain wildcards: IGHA
Sequence dropped because contain wildcards: IGHE
Sequence dropped because contain wildcards: IGHG1A
Sequence dropped because contain wildcards: IGHG1B
Sequence dropped because contain wildcards: IGHG2B
Sequence dropped because contain wildcards: IGHG2C
Sequence dropped because contain wildcards: IGHM
```
My question is what is correct format of C region?

2.The second error is "CDR3 does not end with W or F, was: CAC / H / CDR3End: 29".I am build Vicugna_pacos library using IMGT reference,does this error affect final performance?

**Comment by mizraelson** [comment, marked as answer]
Hi, do your C gene sequences contain N nucleotides? MiXCR requires unambiguity of all nucleotides while building the reference.
The second message is a warning and should not affect the performance.

**Reply by bigcat1001** [reply]
Yes,my C genes contain N,thanks for your help.I will try to modify it.
